packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -2,32 +2,38 @@
 
 import program from 'commander';
 import path from 'path';
-import { listen, get } from '../request';
+import { spawnSync } from 'child_process';
+import fs from 'fs-extra';
+import { listen, get, uploadFile } from '../request';
 import { route } from '../router';
 import { tunaMirrorURI } from '../config';
 import { ora } from '../utils';
 
 async function install(name: string, opts: any): Promise<void> {
-  const spinner = ora();
-  spinner.start(`fetching package info ${name}`);
+  if (name.startsWith('./') || path.isAbsolute(name)) {
```

Comment:
How about another relative path just like ../?

Reply:
I replace it to `.`.

Reply:
How to replace it and where?